Framework/Algorithms/test/CreateTransmissionWorkspace2Test.h

```diff
@@ -363,6 +363,30 @@ class CreateTransmissionWorkspace2Test : public CxxTest::TestSuite {
     TS_ASSERT_DELTA(outLam->x(0)[3], 2.0924, 0.0001);
   }
 
+  void test_two_transmission_runs_stitch_ScaleRHSWorkspace() {
+
+    CreateTransmissionWorkspace2 alg;
+    alg.setChild(true);
+    alg.initialize();
+    alg.setProperty("FirstTransmissionRun", m_multiDetectorWS);
```

Comment:
I'm not sure how this test is really going to be testing your new property? The lhs and rhs workspaces are identical?